tests/phpunit/Constraint/Constraints/ShapeConstraintTest.php

```diff
@@ -69,10 +70,10 @@ public function testCheckConstraint_shape_constraint_missing_property() {
 			->with( $this->equalTo( 'Foo' ) )
 			->will( $this->returnValue( false ) );
 
-		$dataValue = $this->getMockBuilder( '\SMWDataValue' )
-			->disableOriginalConstructor()
-			->setMethods( [ 'addError', 'getCallable' ] )
-			->getMockForAbstractClass();
+		$dataValue = $this->createMock( SMWDataValue::class );
+		$dataValue->expects( $this->any() )
```

Comment:
Isn't `->expects( $this->any() )` the default behaviour?

Reply:
Yeah, calling `->method()` (via a magic method) without setting up expects would work too.